src/redux/actions/student_view_actions.js

```diff
@@ -0,0 +1,17 @@
+export function changeTutorsAction(tutors) {
+  return {
+    type: 'CHANGE_ONLINE_TUTORS',
+    payload: tutors
```

Comment:
you should probably make tutors an object... so like:
```
{
  payload: { tutors }
}
```

Reply:
Tutors is actually a list, so no need to make it an object

Reply:
I gotcha. But still you should make it of the form 
```
{
  payload: {
        tutors: tutorList
  }
}
```

The reason being that this way you can index into it as `payload.tutors[0]` instead of just indexing in as `payload[0]` which is more ambiguous as to what those values are.

Reply:
Gotcha